debezium-connector-oracle/pom.xml

```diff
@@ -470,6 +482,80 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>${version.failsafe.plugin}</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <isAssemblyProfileActive>true</isAssemblyProfileActive>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
```

Comment:
No, it's not a leftover. When building the assembly package at the time of the release the REST Extension tests should be run. This requires enabling the tests with that system property. I am not sure if we can tests this (on the Test Farm, as I suggested on the chat) because it won't work on GHA here because of too limited resources to run Testcontainers Oracle DB instance and the Maven Docker Plugin one at the same time.